weblate_web/static/vendor/simpleSlider.min.css

```diff
@@ -0,0 +1,10 @@
+@charset "UTF-8";
+/*!
+ * SimpleSlider v1.9.0
+ * Simple responsive slider created in pure javascript.

```

Comment:
```suggestion
 * Simple responsive slider created in pure JavaScript.

```

Reply:
This is vendored library, if you wan to make fixes, please do that upstream: https://github.com/michu2k/SimpleSlider